examples/plugins/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
```

Comment:
Rather than having an examples/plugins/ directory, shouldn't we just have plugins/ or letsencrypt/plugins/, and move the apache and standalone plugins into that directory?  And then perhaps include a dummy/example plugin there, if the real ones are too complicated to serve as good examples?